toolchain/parse/typed_nodes.h

```diff
@@ -272,7 +284,7 @@ struct PackageDecl {
   Lex::SemiTokenIndex token;
 };
 
-// `import TheirPackage library "TheirLibrary";`
+// `import [TheirPackage] [library "TheirLibrary"] [inline "code"];`
```

Comment:
Should `library` and `inline` be mutually exclusive? Note, not really caring if the struct reflects it (that may be inconvenient), just the way it's documented.

```suggestion
// `import [TheirPackage] [library "TheirLibrary" | inline "code"];`
```